### ISSUE TITLE
Stumbling block on good first issue

### ISSUE THREAD

**Issue by tvanderpol** [rbspy/rbspy#132]
I figured I'd try my hand on https://github.com/rbspy/rbspy/issues/128 .

I used https://www.rustup.rs to make sure I had the correct version of rust / cargo and tried to run `cargo test` to make sure things were working properly.

I got 477 errors of the type:

```
error: unions are unstable and possibly buggy (see issue #32836)
    --> ruby-structs/src/ruby_2_5_0_rc1.rs:3641:1
     |
3641 |   pub union rb_iseq_struct__bindgen_ty_1 {
     |  _^ starting here...
3642 | |     pub compile_data: *mut iseq_compile_data,
3643 | |     pub loader: rb_iseq_struct__bindgen_ty_1__bindgen_ty_1,
3644 | |     pub trace_events: rb_event_flag_t,
3645 | |     _bindgen_union_align: [u64; 2usize],
3646 | | }
     | |_^ ...ending here
```

This refers me to [this rust issue](https://github.com/rust-lang/rust/issues/32836). And this is where I'm too far out of my depth to keep going.

I could obviously be doing something wrong but I figured I'd flag it here as first issues especially are likely to run aground on this sort of thing.

**Comment by jvns** [comment]
What version of Rust are you using? (`rustc --version`?)

My best guess is that maybe you used rustup to install Rust, but that there's something going on with your PATH where cargo is actually trying to run a different rustc you have installed that's older.

I double checked and I'm definitely able to compile rbspy from scratch with the latest Rust (1.25) that I got by running `rustup update stable`.

**Comment by tvanderpol** [comment]
I tried to check this on my laptop but all went fine. I'll check on the desktop I did this tomorrow - maybe I can work out what went wrong and update this before closing.

**Comment by tvanderpol** [comment]
Ok having had a poke at my installation on this machine - something weird was going on. I did use rustup, I did get my path right (It was in ~/.cargo/bin) and I did try to update. However, for some reason the version I ended up with was from 2017-04 somewhere (I didn't note it, sorry). Doing a full uninstall / reinstall did get me the currently stable rust version 1.25.

So 100% nothing to do with rbspy!
